python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -192,6 +232,10 @@ def _report_current_usage(
                 f"{cur_usage.object_store_memory_str()}/"
                 f"{limits.object_store_memory_str()} object_store_memory"
             )
+        if self._output_info:
+            self._output_info.set_description(
+                f"output: {len(self._output_node.outqueue)} queued"
```

Comment:
Not sure if it's possible right now, but it'd be nice to list the amount of object store memory queued.

Reply:
Yeah, we could do this for each operator. Currently, we show it globally in the global progress bar only.